Libplanet.Net/Consensus/ConsensusReactor.cs

```diff
@@ -37,7 +37,8 @@ public class ConsensusReactor<T> : IReactor
             _consensusTransport = consensusTransport;
             _validators = validators;
             _validatorPeers = validatorPeers?
-                .Where(peer => !peer.PublicKey.Equals(privateKey.PublicKey)).ToImmutableHashSet();
+                .Where(peer => !peer.PublicKey.Equals(privateKey.PublicKey)).ToImmutableHashSet()
```

Comment:
Now, `BroadcastMessage` sends messages to self anyway, instead of  excluding self from here, how about remove the code that adds self  in `BroadcastMessage` [#L107](https://github.com/planetarium/libplanet/pull/2086/files#diff-be45a2ea18eeeb43a9ddca740fd2581952b2b148cbc5b79856771caef762de98R107).

Reply:
23f3d69de84051e2abaa537c64ff052941ee6df8 amended!